libs/pika/schedulers/include/pika/schedulers/shared_priority_queue_scheduler.hpp

```diff
@@ -783,8 +782,7 @@ namespace pika::threads {
             execution::thread_schedule_hint schedulehint, bool allow_fallback,
             execution::thread_priority priority = execution::thread_priority::normal) override
         {
-            schedule_work(thrd, schedulehint, allow_fallback, false,
-                priority = execution::thread_priority::normal);
```

Comment:
No action required, just FYI: `nvc++` actually complained about this assignment, so this is also fixed on #606.